dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:stretch-slim
+
+MAINTAINER Eric Goller
+
+RUN apt-get update -qq
+RUN apt-get install --no-install-recommends -y git python-setuptools python-pip python-dev python3-dev build-essential python-wheel xvfb xauth curl locales
+RUN locale-gen C.UTF-8 && /usr/sbin/update-locale LANG=C.UTF-8
+ENV LANG=C.UTF-8 LANGUAGE=C.UTF-8 LC_ALL=C.UTF-8
+
+WORKDIR /hamster-gtk
+RUN apt-get install --no-install-recommends -y gir1.2-pango-1.0 gir1.2-gtk-3.0 libglib2.0-dev libgtk-3-dev python-gi python3-gi python-cairo python-gi-cairo
+RUN git clone https://github.com/projecthamster/hamster-gtk.git
```

Comment:
Would it be possible to build the docker image from pwd instead of from the cloned repository? Otherwise, we will not be able to test changes in this on Travis easily.